modules/book-class.js

```diff
@@ -0,0 +1,7 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
- [ ] Please note that Linters analyzes source code to flag programming errors, bugs, stylistic errors, and suspicious constructs and it will make sure that your code looks clean, organized, and readable,  Kindly Remove the linters disable commands and try to fix the errors 🙏 

Tip : 
I noticed that you are having an issue with the default export linter rule, in this case, you can export your class/function as default like this : 

```JS
export default class Book {
  constructor(title, author) {
    this.title = title;
    this.author = author;
  }
}
```

and import it in `index.js` like this 

```JS
//instead of import {Book} from './modules/book-class.js';
import Book from './modules/book-class.js';
```

🛑 **_you can apply this to all of your export/import commands to avoid the linters errors_**